app/assets/stylesheets/application.tailwind.css

```diff
@@ -0,0 +1,53 @@
+/* stylelint-disable */
+
```

Comment:
[OPTIONAL] You did good job. But, somehow, your design doesn't meet the original design. So, I would suggest you to maybe give some attention for styling.